README.md

```diff
@@ -23,7 +23,7 @@ Some example CLI calls:
 Get the CLI form:
 
 ```shell
-$ ./some_python_venv/bin/energyplus_idd_idf --help
+$ energyplus_idd_idf --help
```

Comment:
User experience would be to either install this into system python and run these binaries from PATH, or install into a virtual environment and activate it.  Either way, the binary should be available by name.